immuneML/reports/data_reports/PDBDataReport.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+from immuneML.reports.encoding_reports.EncodingReport import EncodingReport
+from plotly.subplots import make_subplots
+
+import pandas as pd
+import plotly.express as px
+from Bio.PDB import PDBParser
+
+from immuneML.data_model.dataset.RepertoireDataset import RepertoireDataset
+from immuneML.data_model.repertoire.Repertoire import Repertoire
+from immuneML.reports.ReportOutput import ReportOutput
+from immuneML.reports.ReportResult import ReportResult
+from immuneML.reports.data_reports.DataReport import DataReport
+from immuneML.util.PathBuilder import PathBuilder
+
+
+class PDBDataReport(EncodingReport):
```

Comment:
this too should have a specific name, there could be many reports for PDBData. something like 3DDistanceHeatmap?